docs/source/gcp/examples/pubsub-snapshots.rst

```diff
@@ -0,0 +1,25 @@
+Pub/Sub - Check if there are expiring Snapshots
+===============================================
+Custodian can check if a Pub/Sub Snapshot expires in a set number of days. Note that the ``notify`` action requires a Pub/Sub topic to be configured.
+
+In the example below, the policy notifies users if there are Snapshots expiring in 2 days.
```

Comment:
given snapshots for pub/sub have a hard time limit on expiration, this example seems less than useful, it would always be firing.